src/CheckoutSdk/Reconcilation/PaymentReportData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Checkout.Common;
+using Checkout.Common.Extensions;
+using System;
+using System.Collections.Generic;
+
+namespace Checkout.Reconciliation
+{
+	public sealed class PaymentReportData : EquatableResource<PaymentReportData>
+	{
+		public string Id { get; set; }
+
+		public double ProcessingCurrency { get; set; }
+
+		public double PayoutCurrency{ get; set; }
+
+		public DateTime RequestedOn{ get; set; }
+
+		public string ChannelName{ get; set; }
+
+		public string Reference{ get; set; }
+
+		public string PaymentMethod{ get; set; }
+
+		public string CardType{ get; set; }
+
+		public string CardCategory{ get; set; }
+
+		public CountryCode IssuerCountry{ get; set; }
+
+		public CountryCode MerchantCountry{ get; set; }
+
+		public string Mid{ get; set; }
+
+		public IList<Action> Actions{ get; set; }
+
+		public override bool EqualExp(PaymentReportData other)
+			=> Id.EqualsNull(other.Id);
+
+		public override int GetHashCode()
```

Comment:
Depending, I don't know how it will behave the class. It will be an mutable object or not?  
    "GetHashCode should not be implemented for mutable types"
For that reason I use the Id, the only property on the object that I know it will not mutate.

*For making the hash, we have to combine the values of the properties using HashCode.Combine, and it has a limitation to combine no more than 8 properties. This class has more of that, what criteria I should use to select the properties? making HasCode.Combine(HasCode.Combine(A,B,..), HasCode.Combine(D. E..)) is not good practice.